tensorflow/core/kernels/mkl_tfconv_op.h

```diff
@@ -37,13 +37,17 @@ limitations under the License.
 using mkldnn::stream;
 
 namespace tensorflow {
+
 #ifdef ENABLE_MKLDNN_V1
 #define ENGINE_CPU engine::kind::cpu
+#define NET_ARGS_PTR &net_args
```

Comment:
`NET_ARGS_PTR` is not used in this file.

Reply:
Yes, you're right. I removed this macro.